src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -50,10 +50,10 @@
      *
      * This argument allows you to set the GQ boundaries. HC expects a list of multiple GQ threshold values. To pass
      * multiple values, you provide them one by one with the argument, as in `-GQB 10 -GQB 20 -GQB 30` and so on. Note
-     * that GQ values are capped at 99 in the GATK.
+     * that GQ values are capped at 99 in the GATK, so values must be integers in the range [1, 99].
```

Comment:
These specify the lower inclusive bounds, so I think 99 is correct (perhaps the old documentation should be updated if this is not clear).  I believe that putting in 100 causes an exception to be thrown, but I'm on my phone so I can't easily check this.